test/stat_tracker_test.rb

```diff
@@ -28,15 +28,27 @@ def test_it_exists
   #   assert_equal './data/game_teams_fixture.csv', @stat_tracker.game_teams_path
   # end
 
-  # Game Statistics Tests - Helper Methods #
+# Game Statistics Tests - Helper Methods #
   def test_it_can_get_total_goals_per_game
     assert_equal 5, @stat_tracker.total_goals_per_game[2012030221]
     assert_equal 3, @stat_tracker.total_goals_per_game[2012030231]
   end
 
-  # Game Statistics Tests - Stat Methods #
+  def test_it_can_get_away_wins
```

Comment:
Is this one by team? I know it can be a helper method, but do we need to pass in the team id?

Reply:
@sarynm12  that's what I was wondering to! The description is a bit confusing: "Percentage of games that a visitor has won (rounded to the nearest 100th)". Does that mean % visitor games won by *all* teams or just one?

Reply:
Nevermind, I take back what I said. They would have had this live this under Team Stats if that were the case.

Reply:
Yeah, it's super confusing! I take it as % visitor wins by team, which would mean we need to pass it a team id. I'll write it down as a clarifying question, though!

Reply:
Oh wait, that's true! So I guess it is by all teams. That seems random, but oh well!

Reply:
No, I don't think it's by team. I think it's just "of the data, for each game, did the away team win?"